resolwe_bio/processes/genome_browser/UCSC_track_hub.yml

```diff
@@ -0,0 +1,75 @@
+# ==============
+# UCSC track hub
+# ==============
+---
+
+- slug: genomebrowser-macs14
+  name: UCSC track hub
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: "resolwebio/chipseq:1.1.0"
+    resources:
+      network: true
+  data_name: "UCSC track hub ({{ name }})"
+  version: 0.0.1
+  type: data:genomebrowser:all:bigwig:macs14
+  category: genomebrowser
+  persistence: CACHED
+  description: >
+    Create UCSC track hub.
+  input:
+    - name: macs
+      label: MACS1.4
+      type: list:data:chipseq:macs14
+    - name: name
+      label: Track hub name
+      type: basic:string
+    - name: email
+      label: Email address
+      type: basic:string
+      required: false
+  output:
+    - name: url_link
+      label: UCSC track hub link
+      type: basic:string
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+      NAME={{ name }}
+      BUILD={{ macs[0].build }}
+      ROOT=${PWD}
+
+      wget http://hgdownload.cse.ucsc.edu/goldenPath/${BUILD}/bigZips/${BUILD}.chrom.sizes
+      re-progress 0.2
+
+      mkdir /web/ucsc/${NAME}
+      mkdir /web/ucsc/${NAME}/${BUILD}
+
+      sample_names="" 
+      {% for mac in macs %}
+        echo {{ mac|name }}
+        if [ {{ mac.build }} != $BUILD ]; then
+          re-error "All selected samples must have the same genome build"
```

Comment:
Error messages were improved in more recent PRs. If necessary, unify this with the approach used by @JenkoB